IgOmeProfiling_pipeline.py

```diff
@@ -174,7 +174,9 @@ def run_pipeline(fastq_path, barcode2samplename_path, samplename2biologicalcondi
                              f'--cv_num_of_splits {cv_num_of_splits}', f'--seed_random_forest {seed_random_forest}',
                              f'--random_forest_seed_configurations {random_forest_seed_configurations}', f'--rank_method {rank_method}', 
                              '--is_run_random_forest_per_bc_sequentially' if is_run_random_forest_per_bc_sequentially else '',
-                             '' if no_rpm_factor else '--no_rpm_factor', '--use_rpm_faa_scanning' if use_rpm_faa_scanning else '',
+                             '' if no_rpm_factor else '--no_rpm_factor',

```

Comment:
If I input the flag `--no_rpm_factor` the value of the variable is false. 
So if it is false I want to use it in the next script too.

Reply:
It is very confusing, if I understand correctly
`no_rpm_factor = false` means `use_rpm_factor = false` and `no_rpm_factor = true` means `use_rpm_factor = true` but logically it should be opossite
`no_rpm_factor = true` means `use_rpm_factor = false`

Reply:
So I will change the action to be true -  when we use the flag  `no_rpm_factor` it will get  `no_rpm_factor = true` and then  `use_rpm_factor = false`